go.sum

```diff
@@ -41,7 +41,6 @@ cloud.google.com/go v0.97.0/go.mod h1:GF7l59pYBVlXQIBLx3a761cZ41F9bBH3JUlihCt2Ud
 cloud.google.com/go v0.98.0/go.mod h1:ua6Ush4NALrHk5QXDWnjvZHN93OuF0HfuEPq9I1X0cM=
```

Comment:
Is it possible to update `go.sum` in a separate PR? I'm just trying to be careful with dependencies so I think we should update `go.mod` and `go.sum` on PRs where it's clearer why they're being updated.

Reply:
Thanks, I can do that.

Reply:
I rolled back the change